CMakeLists.txt

```diff
@@ -157,14 +173,31 @@ if(NOT PYTHON_MODULE_DIR)
 		OUTPUT_STRIP_TRAILING_WHITESPACE
 	)
 endif(NOT PYTHON_MODULE_DIR)
-message("PYTHON_MODULE_DIR is ${PYTHON_MODULE_DIR}")
+message(STATUS "Python modules will be installed to ${PYTHON_MODULE_DIR}")
 
 foreach(lib ${SPT3G_LIBRARIES})
 	INSTALL(TARGETS ${lib} EXPORT ${PROJECT_NAME}Config)
 endforeach(lib ${SPT3G_LIBRARIES})
 
 # The python module
-INSTALL(DIRECTORY ${SPT3G_LIBRARY_DIR} DESTINATION ${PYTHON_MODULE_DIR})
+INSTALL(FILES ${CMAKE_SOURCE_DIR}/cmake/init.py DESTINATION ${PYTHON_MODULE_DIR}/spt3g RENAME __init__.py)
+foreach(lib ${SPT3G_LIBRARIES})
+	if("${lib}" STREQUAL "spt3g")
+		continue() # skip phony library
+	endif("${lib}" STREQUAL "spt3g")
+
+	if(EXISTS ${CMAKE_SOURCE_DIR}/${lib}/python)
```

Comment:
Some of the libraries in the private repository don’t have a `python` us directory.  There should be a function that sets that directory and creates the symlinks, maybe the install code can be part of that function instead?